src/test/java/com/hocs/test/pages/data_input/DataInput.java

```diff
@@ -206,15 +215,52 @@ public void enterReferenceText() {
         referenceTextField.sendKeys("");
     }
 
-    public void fillAllMandatoryFields() {
+    public void fillAllMandatoryCorrespondenceFields() {
+        enterDayOfCorrespondenceSent(todayPlusNDaysGetDay(-2));
+        enterMonthOfCorrespondenceSent(todayPlusNDaysGetMonth(-2));
+        enterYearOfCorrespondenceSent(todayPlusNDaysGetYear(-2));
+        enterDayOfCorrespondenceReceived(getCurrentDay());
+        enterMonthOfCorrespondenceReceived(getCurrentMonth());
+        enterYearOfCorrespondenceReceived(getCurrentYear());
+        emailOriginalChannelRadioButton.click();
+        selectN10ResponseNoRadioButton();
+    }
+
+    public void fillAllMandatoryCorrespondenceFieldsWithCopyToNumberTenYes() {
         enterDayOfCorrespondenceSent(todayPlusNDaysGetDay(-2));
         enterMonthOfCorrespondenceSent(todayPlusNDaysGetMonth(-2));
         enterYearOfCorrespondenceSent(todayPlusNDaysGetYear(-2));
         enterDayOfCorrespondenceReceived(getCurrentDay());
         enterMonthOfCorrespondenceReceived(getCurrentMonth());
         enterYearOfCorrespondenceReceived(getCurrentYear());
         emailOriginalChannelRadioButton.click();
+        selectN10ResponseYesRadioButton();
+    }
+
+    public void getToAddMemberOfParliamentPrerequisites() {
```

Comment:
getToAddMemberOfParliamentPrerequisites and getToRecordCorrespondentDetailsPrerequisites seem to do the same thing ?

Reply:
Record Correspondent Details - Member of the public
Add Member of Parliament - Member of Parliament